composer.lock

```diff
@@ -4,7 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
-    "content-hash": "e8806838d7235c3d2ab386293836949f",
+    "hash": "3749ec07f0cd5aa8a58d5252a2c19ca4",
+    "content-hash": "dbdc24fc713f6a6da88dcdff23201f1b",
```

Comment:
please update your composer version to use composer 1.3, instead of reverting the lock file to things created by an older version (different date format and the old ``hash``), and then regenerate the lock with ``composer update --lock``